packages/ketcher-react/src/script/editor/tool/rotate-controller.ts

```diff
@@ -27,15 +31,15 @@ const RIGHT_ARROW_PATH =
 
 class RotateController {
   isRotating!: boolean
+  handleCenter!: Vec2
   private editor: Editor
   private rotateTool: RotateTool
   private originalCenter!: Vec2
   private normalizedCenterInitialHandleVec!: Vec2
-  private handleCenter!: Vec2
   private initialRadius!: number
   private isMovingCenter!: boolean
 
-  private handle?: RaphaelElement
+  handle?: RaphaelElement
```

Comment:
it looks like making this property public is not very required, so we can make a new public method with a stricter interface, let's say you can implement scrolling logic inside this method even without exposing internal properties.

Reply:
Such a great idea, thank you!